bot.js

```diff
@@ -7,61 +7,50 @@ const bot = new Discord.Client();
 const token = process.env.TOKEN;
 const prefix = process.env.PREFIX;
 
-bot.on('ready', () => {
-	console.log(`Logged in as ${bot.user.username}!`);
-});
-
+bot.on('ready', () => console.log(`Logged in as ${bot.user.username}!`));
 bot.on('message', msg => {
-	console.log(`RECV: ${msg.author.tag}@${msg.channel.name} ${msg.content}`);
```

Comment:
Can we get this or whatever equivalent back?

Reply:
Why log stuff though?

Reply:
it helps debugging :)